### ISSUE TITLE
[v3] Errors occured while trying to save

### ISSUE THREAD

**Issue by DujaOSM** [openstreetmap/iD#7155]
Trying to upload a rather large changeset with v3 (version ef3ccca in the status bar, whatever that means), I got an "Errors occured while trying to save", with no hint what went wrong. Now, I'm not sure if I hit some API limit (I've split a highway and disturbed a few large route relations) or if I discovered a bug in iD. Here's an osmChange file I downloaded:
[changes.osc.xml.zip](https://github.com/openstreetmap/iD/files/3995129/changes.osc.xml.zip)

**Comment by mmd-osm** [comment]
API limit would be a maximum of 10'000 changes per changeset. I think your changeset is smaller than that.

**Comment by DujaOSM** [comment]
When I undid the operations multiple times, iD started validating against conflicts, which took quite a while (so it might be related with #3155 - Conflict resolution at save time is slow). I managed to save about half of the changeset after I exited and re-entered iD and restored the changes (redo is not available after unsuccessful saving so those are lost for good). 

Still, I would be interested how I can debug the issue when it happens on an API level as here. Further, is there a tool other than JOSM (that I don't fancy exploring) that I can feed the osmChange file and try to upload it?

**Comment by mmd-osm** [comment]
Well, simply trying to upload will not work here because of the conflicts, in which case the api will reject this file. You need some tool to do the conflict resolution, like josm.

**Comment by DujaOSM** [comment]
By now, I salvaged what I could from the original changeset, and recreated the rest manually. Oh well, half an hour of work or so lost, my insurance will reimburse. :)

I'm not sure what caused the original problem: it was not #3155 in its pure form, when I tried to save I received "Errors occurred" immediately. 

Anyway, I realize this is a messy report, since it's hard to reconstruct what went wrong. If devs want to investigate whether iD  generated a bad changeset, it's attached; otherwise, I don't mind if this issue is closed.
